source/release-notes.txt

```diff
@@ -10,6 +10,36 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.45.1
+----------------
+
+*Released January 15, 2024*
+
+New Features:
+
+- Adds welcome page (:issue:`COMPASS-8657`)
+- Adds onFailToLoadConnections callback (:issue:`COMPASS-8657`)
+
+Bug Fixes:
+
+- Send a nonce in oidc request by default (:issue:`COMPASS-8588`)
+- Don't cut off list view value editor tooltips (:issue:`COMPASS-8581`)
+- Fixes element key editor tooltip and revert not updating validity 
+  (:issue:`COMPASS-8586`)
+- Don't fallback to zero if the count cannot be determined
```

Comment:
[question for tech reviewer]

Does this have an associated issue #? I don't see one listed here: https://github.com/mongodb-js/compass/releases/tag/v1.45.1 Should we consider removing this? Thanks!

Reply:
added [COMPASS-8841](https://jira.mongodb.org/browse/COMPASS-8841) for this